### DISCUSSION TITLE
Vetting

### DISCUSSION THREAD

**Discussion by Batnonhrattata** [landgreen/n-gon#247] [General]
Hello. I would like to know if I can be vetted. I've been playing for 2 yrs (close enough) and have communicated with Mr. Landgreen himself. and AFA personnel. Just wondering.

**Comment by kgurchiek** [comment]
What do you mean?

**Comment by Batnonhrattata** [comment]
i just wanna be vetted

On Wed, May 8, 2024 at 5:54 PM Cornbread2100 ***@***.***>
wrote:

> What do you mean?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/landgreen/n-gon/discussions/247#discussioncomment-9360737>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYCHGEPM7SRTEU5Z7ADLMA3ZBKNIBAVCNFSM6AAAAABHNHUGJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TGNRQG4ZTO>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Julian Crow Sipple*

**Comment by kgurchiek** [comment]
I don't know what you mean by that, what do you mean?

**Comment by Batnonhrattata** [comment]
like you know how in mc vets are people whove been playing for a long time.

On Thu, May 9, 2024 at 9:41 AM Cornbread2100 ***@***.***>
wrote:

> I don't know what you mean by that, what do you mean?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/landgreen/n-gon/discussions/247#discussioncomment-9368994>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYCHGEIF4NR5DDK46SZTRMDZBN4GXAVCNFSM6AAAAABHNHUGJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TGNRYHE4TI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Julian Crow Sipple*

**Comment by kgurchiek** [comment]
I've never heard of any official way veteran mc players are recognized, generally in both mc's community and this one you gain reputation and respect naturally over time by being a major member of the community, there isn't any single point where you're officially "vetted".

**Comment by Batnonhrattata** [comment]
OK thx dawg

On Thu, May 9, 2024 at 4:51 PM Cornbread2100 ***@***.***>
wrote:

> I've never heard of any official way veteran mc players are recognized,
> generally in both mc's community and this one you gain reputation and
> respect naturally over time by being a major member of the community, there
> isn't any single point where you're officially "vetted".
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/landgreen/n-gon/discussions/247#discussioncomment-9375098>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYCHGEMH4WTFDTKPETCX2ALZBPOVLAVCNFSM6AAAAABHNHUGJWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TGNZVGA4TQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Julian Crow Sipple*

**Comment by 73rb** [comment]
can you make a new link it got blocked
or a download

**Comment by AnnonymousNerd87** [comment]
I already made a post for this, you can just download the zip from the
github, to open the game open index.html

**Reply by 73rb** [reply]
im knew to girhub sorry idk what u talking about 😭

**Reply by kgurchiek** [reply]
You can download the latest release of the game here: https://codeload.github.com/landgreen/n-gon/zip/refs/heads/master. Extract the downloaded zip file, open the folder, and open `index.html`

**Reply by 73rb** [reply]
thanks i got it

**Comment by AnnonymousNerd87** [comment]
click code, then click download zip, then open the file named index.html

**Comment by AnnonymousNerd87** [comment]
yeah that works too
